tardis/montecarlo/montecarlo_numba/tests/test_packet.py

```diff
@@ -31,6 +29,8 @@ def model():
     return numba_interface.NumbaModel(
         r_inner=np.array([6.912e14, 8.64e14], dtype=np.float64),
         r_outer=np.array([8.64e14, 1.0368e15], dtype=np.float64),
+        v_inner=np.array([6.912e14, 8.64e14], dtype=np.float64),
+        v_outer=np.array([8.64e14, 1.0368e15], dtype=np.float64),
```

Comment:
Since these match the radius values, I assume they do not affect the tests. In which case it would be good to put nonsense values in like -1 so it's clear they are not significant.